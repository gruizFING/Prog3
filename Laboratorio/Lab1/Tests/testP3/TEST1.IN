TEST DE VARIABLES
InicializoLasVariablesProbandoConMuchosExpacios
VAR =12
VAR    =0.3
VAR   =    2.0
VAR var1= 333
VAR var2   = 1000
VAR    var3=78
VAR var4   =9
VAR var5=99
VAR var6  = 88
VAR var7 =78
--Cambio el valor de la var2 evaluando antes y despues del cambio--
EVAL var2
VAR var2=333
EVAL var2
con nombre vacio 
VAR =89
VAR      =     99
VAR   =77
VAR var8= 3
VAR var9  =98
VAR var10 = 0000.58
VAR var11 = 48
EVAL var9
VAR var9 =789
VAR var8 = 12.0091
EVAL var8
ProbandoEvaluar Expresiones
EVAL var9
EVAL var1
EVAL var6
EVAL var3
con nombre vacio y diccionario lleno
VAR =89
VAR      =     99
VAR   =77
simbolos raros "#$%45
EVAL var10
EVAL var2
EVAL var4
EVAL var8
EVAL var5
EVAL var7
Cambio el valor muchas veces y hago eval...
SALIR