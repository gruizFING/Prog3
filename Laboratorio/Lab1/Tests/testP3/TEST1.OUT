Bienvenido a el mini-sistema:
> Comando no reconocido.
> Comando no reconocido.
> ERROR: Nombre de variable vacio.
> ERROR: Nombre de variable vacio.
> ERROR: Nombre de variable vacio.
> OK. var1 agregada.
> OK. var2 agregada.
> OK. var3 agregada.
> OK. var4 agregada.
> OK. var5 agregada.
> OK. var6 agregada.
> OK. var7 agregada.
> Comando no reconocido.
> OK. 1000.00
> OK. var2 agregada.
> OK. 333.00
> Comando no reconocido.
> ERROR: Nombre de variable vacio.
> ERROR: Nombre de variable vacio.
> ERROR: Nombre de variable vacio.
> OK. var8 agregada.
> OK. var9 agregada.
> OK. var10 agregada.
> ERROR: Se alcanzo la maxima cantidad de variables.
> OK. 98.00
> OK. var9 agregada.
> OK. var8 agregada.
> OK. 12.01
> Comando no reconocido.
> OK. 789.00
> OK. 333.00
> OK. 88.00
> OK. 78.00
> Comando no reconocido.
> ERROR: Nombre de variable vacio.
> ERROR: Nombre de variable vacio.
> ERROR: Nombre de variable vacio.
> Comando no reconocido.
> OK. 0.58
> OK. 333.00
> OK. 9.00
> OK. 12.01
> OK. 99.00
> OK. 78.00
> Comando no reconocido.
> 